### ISSUE TITLE
Keyboard shortcut to jump to next first frame of a track

### ISSUE THREAD

**Issue by manybodycpa** [opencv/cvat#2051]
Would it be possible to have a shortcut to quickly navigate through the different tracks? For example, I have track 1 from frames 50-100, track 2 from 75-125 and track 3 from 150 to 200. With the shortcut, I would press the corresponding key once to go from frame 0 directly to 50, a second time to jump to frame 75, and a third time to jump to frame 150. For example, this would be very useful to loop through tracks and eliminate false positive detections or check immutable labels of entire tracks.

**Comment by bsekachev** [comment]
I can suggest adding shortcuts to jump to the first/last frame of a track under mouse.
Going to the first frame of a next track looks at least not obvious (we can have a lot of tracks on different frames)

**Comment by manybodycpa** [comment]
Good point. Could we jump to the next frame that contains the beginning (first box) of one or more tracks?

**Comment by Sk05062003** [comment]
I worked around a bit on this issue. I am able to jump to initial frames of each track individually, through keyboard shortcuts. Please assign me this issue so that I can further work on it.

Thanks
GSOC'24 applicant

**Comment by rukundob451** [comment]
Hi all,
Has there been any progress on implementing the keyboard shortcut for jumping to the next track frame? 
Thanks,
Benjamin.

**Comment by rayenmk** [comment]
Hi, I’m Rayen, a GSoC applicant interested in contributing.

I would like to work on this issue. I’ll start investigating and propose a fix soon.

Thanks!

**Comment by AnanthKumar1810** [comment]
Hi, I'm Ananth . is this still open? I'd like to work on it.
